components/AppNavigation.js

```diff
@@ -58,7 +58,12 @@ const AppNavigation = () => {
                 <Drawer.Screen name="HealthProfessional" component={HealthProfessionalScreen} />
                 <Drawer.Screen name="SymptomChecklistScreen" component={SymptomChecklistScreen} />
                 <Drawer.Screen name="NotificationScreen" component={NotificationScreen}/>
+                <Drawer.Screen name="UserProfileScreen" component={UserProfileScreen}/>
+<<<<<<< HEAD
```

Comment:
Merge conflict here,

Tip: make sure to `git pull` before you git commit and git push, otherwise merge conflicts will happen without you knowing. Git pull will force git to make the user fix the conflicts before merging.